Dockerfile

```diff
@@ -18,21 +18,20 @@ ENV WORKDIR /opt/topological_inventory-api/
 ENV RAILS_ROOT $WORKDIR
 WORKDIR $WORKDIR
 
-COPY Gemfile $WORKDIR
+COPY . $WORKDIR
+COPY docker-assets/entrypoint /usr/bin
+
+RUN chgrp -R 0 $WORKDIR && \
+    chmod -R g=u $WORKDIR
```

Comment:
Why move this up? Could this move cause an issue if the gems are installed in `$WORKDIR`?